### PR TITLE
Complete list of smoke tests

### PR DESCRIPTION
**Pull request by giorgiosironi**
Looking at PathMediaTypeMapper it seems we are missing some API checks after every deploy.

### CODE REVIEW

File: smoke_tests.sh [hunk 1]
```diff
@@ -11,13 +11,17 @@ echo "Homepage"
 [ $(curl --write-out %{http_code} --silent --output /dev/null "$hostname") == 200 ]
 
 echo "APIs"
+[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.annual-report-list+json; version=1" "${hostname}/blog-articles") == 200 ]
```
Comment (nlisgo) [inline smoke_tests.sh]: Incorrect path: should be /annual-reports

File: smoke_tests.sh [hunk 2]
```diff
@@ -11,13 +11,17 @@ echo "Homepage"
...
 [ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.blog-article-list+json; version=1" "${hostname}/blog-articles") == 200 ]
 [ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.collection-list+json; version=1" "${hostname}/collections") == 200 ]
+[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.community-list+json; version=1" "${hostname}/covers") == 200 ]
```
Comment (nlisgo) [inline smoke_tests.sh]: Incorrect path

File: smoke_tests.sh [hunk 3]
```diff
@@ -11,13 +11,17 @@ echo "Homepage"
...
 [ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.cover-list+json; version=1" "${hostname}/covers") == 200 ]
 [ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.event-list+json; version=1" "${hostname}/events") == 200 ]
 [ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.interview-list+json; version=1" "${hostname}/interviews") == 200 ]
+[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.job-advert-list+json; version=1" "${hostname}/interviews") == 200 ]
```
Comment (nlisgo) [inline smoke_tests.sh]: Incorrect path

File: smoke_tests.sh [hunk 4]
```diff
@@ -11,13 +11,17 @@ echo "Homepage"
...
 [ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.labs-post-list+json; version=1" "${hostname}/labs-posts") == 200 ]
 [ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.person-list+json; version=1" "${hostname}/people") == 200 ]
+[ $(curl --write-out %{http_code} --silent --output /dev/null -H "Accept: application/vnd.elife.press-package-list+json; version=1" "${hostname}/people") == 200 ]
```
Comment (nlisgo) [inline smoke_tests.sh]: Incorrect path

Comment (giorgiosironi) [reply smoke_tests.sh]: Copy&paste :)

**Comment by giorgiosironi** [comment]
Weirdly the wrong smoke tests passed:
```
17:25:54 [34.224.22.179] out: ++ curl --write-out '%{http_code}' --silent --output /dev/null -H 'Accept: application/vnd.elife.annual-report-list+json; version=1' ci--journal-cms.elifesciences.org/blog-articles
17:25:54 [34.224.22.179] out: + '[' 200 == 200 ']'
```
Maybe I should just remove the `Accept` headers, which also need `version` to be updated, and just target the paths? That would also make for a clean `for` loop rather than rewriting all the curl commands.

**Comment by giorgiosironi** [comment]
Removing all `Accept` for simplicity, won't need to track API versions. Dependency on particular `Accept` headers should not be tested with smoke tests but with PHPUnit.

**Comment by giorgiosironi** [comment]
Let's :shipit:

**Comment by giorgiosironi** [comment]
However, `continuumtest` is currently being switched to a test branch so can't merge.

**Comment by nlisgo** [comment]
Merged in latest changes in `develop`

**Pull request merged**